### DISCUSSION TITLE
Question:

### DISCUSSION THREAD

**Discussion by ghost** [FunkinCrew/Funkin#1507] [Q&A] [answered]
## What is your question?
Why is the dad purple? Is he the man behind the slaughter? So many unanswered questions!!!!11

**Comment by Windows1Door2Daniel** [comment]
maybe its his parents and ancestors are purple too

can be

1. black eyes with red pupil
2. hair with horns
3. purple skin
all gf family life phases
maybe can happen that

**Comment by ghost** [comment]
Maybe the mom and dad adopted the gf

**Comment by SilverT4** [comment, marked as answer]
I believe Daddy Dearest (and Mommy Must Murder) are both demons.
